src/main/java/TaskForLesson4/ChatController.java

```diff
@@ -1,4 +1,32 @@
 package TaskForLesson4;
 
+import javafx.scene.control.TextField;
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyEvent;
+
+import java.util.ArrayList;
+
 public class ChatController {
+    public TextField output;        //поле Чата
+    public TextField input;        //поел ввода текста сообщения
+    private final ArrayList<String> TextChat = new ArrayList<>();
+
+    public void inputEnter() {   //кнопка ввода текста
+        if (!input.getText().isEmpty()) {
+            TextChat.add(input.getText());
+            output.setText(String.format("%s %n", TextChat));
+            input.clear();
+        }
+
+    }
+
+    public void Enter(KeyEvent keyEvent) {                              // НЕ РАБОТАЕТ
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.